File: src/backend/guild-server/src/main/java/com/asyncgate/guild_server/config/FeignClientConfig.java
package com.asyncgate.guild_server.config;

import feign.Logger;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
@EnableFeignClients(basePackages = "com.asyncgate.guild_server.client")
public class FeignClientConfig {

    @Bean
    public Logger.Level feignLoggerLevel() {
        return Logger.Level.FULL;
    }
}

File: src/backend/chat-server/src/main/kotlin/com/asyncgate/chat_server/config/FeignClientConfig.kt
package com.asyncgate.chat_server.config

import feign.Logger
import org.springframework.cloud.openfeign.EnableFeignClients
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
@EnableFeignClients
class FeignClientConfig {

    @Bean
    fun feignLoggerLevel(): Logger.Level {
        return Logger.Level.FULL
    }
}